style/javascript/sample.js

```diff
@@ -0,0 +1,7 @@
+object = {spacing: false}
```

Comment:
Er. Why not put spaces around the curly braces? I like  those spaces, because it makes it easier to read. Same reason I like 'em in Ruby.